.github/workflows/sycl_update_gpu_driver.yml

```diff
@@ -0,0 +1,58 @@
+name: Update GPU driver
+
+on:
+  schedule:
+    - cron:  '0 3 * * 2'
+  workflow_dispatch:
+
+jobs:
+  update_driver_linux:
+    runs-on: ubuntu-latest
+    if: github.repository == 'intel/llvm'
+    steps:
+    - uses: actions/checkout@v2
+    - name: Update dependencies file
+      run: |
+        version="$(python3 devops/scripts/update_drivers.py linux)"
+        echo 'NEW_DRIVER_VERSION='$version >> $GITHUB_ENV
+    - name: Create Pull Request
+      env:
+        BRANCH: ci/update_gpu_driver-linux
+        LLVMBOT_TOKEN: ${{ secrets.LLVM_MAIN_SYNC_BBSYCL_TOKEN }}
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      run: |
+        cd $GITHUB_WORKSPACE
+        # Set fake identity to fulfil git requirements
+        git config --global user.name "iclsrc"
+        git config --global user.email "ia.compiler.tools.git@intel.com"
+        git checkout -B $BRANCH
+        git add -u
+        git commit -m "[GHA] Uplift GPU RT version for Linux CI"
+        git push https://$LLVMBOT_TOKEN@github.com/${{ github.repository }} ${BRANCH} --force
+        gh pr create --title "[GHA] Uplift GPU RT version for Linux CI" --body "Uplift GPU RT version for Linux to $NEW_DRIVER_VERSION" --assignee @intel/dpcpp-devops-reviewers
+
+  update_driver_linux_staging:
+    runs-on: ubuntu-latest
+    if: github.repository == 'intel/llvm'
+    steps:
+    - uses: actions/checkout@v2
+    - name: Update dependencies file
+      run: |
+        version="$(python3 devops/scripts/update_drivers.py linux_staging)"
+        echo 'NEW_DRIVER_VERSION='$version >> $GITHUB_ENV
+    - name: Create Pull Request
+      env:
+        BRANCH: ci/update_gpu_driver-linux_staging
+        LLVMBOT_TOKEN: ${{ secrets.LLVM_MAIN_SYNC_BBSYCL_TOKEN }}
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      run: |
+        cd $GITHUB_WORKSPACE
+        # Set fake identity to fulfil git requirements
+        git config --global user.name "iclsrc"
+        git config --global user.email "ia.compiler.tools.git@intel.com"
+        git checkout -B $BRANCH
+        git add -u
+        git commit -m "[GHA] Uplift GPU RT version for Nightly runs"
+        git push https://$LLVMBOT_TOKEN@github.com/${{ github.repository }} ${BRANCH} --force
```

Comment:
@bader ,
I was unable to test this on my own repo as it is written here. No $LLVMBOT_TOKEN.  Instead when testing I used a simple `git push origin ${BRANCH} --force` and that worked.  
Here I'm leaving it intact as written (with the addition of the --force flag). Let me know if you think we should keep/change.

Reply:
> @bader , I was unable to test this on my own repo as it is written here. No $LLVMBOT_TOKEN. 
@tfzhu, could you help with local testing, please?

> Instead when testing I used a simple `git push origin ${BRANCH} --force` and that worked. Here I'm leaving it intact as written (with the addition of the --force flag). Let me know if you think we should keep/change.

I prefer avoiding force-push. Most likely existing branch will be already linked to some PR.
One way to avoid collisions with existing PRs is to add $version value to the branch name. If such branch already exists, it means that there is already PR updating the driver to $version and there is no need in creating a new PR.

Reply:
I added the version number to the branch names and removed the --force flag from the push.